CONTRIBUTING.md

```diff
@@ -28,11 +29,16 @@ All contributors are expected to follow the [Rust Code of Conduct].
 
 ## Getting started
 
-High level approach:
+**Note:** If this is your first time contributing to Clippy, you should
+first read the [Basics docs](doc/basics.md). It will help you get set up
+for working on Clippy. There's also more information about building Clippy
+[below](#how-to-build-clippy).
```

Comment:
```suggestion
**Note:** If this is your first time contributing to Clippy, you should
first read the [Basics docs](doc/basics.md). It will help you get set up
for working on Clippy. 
```

Keep important notes as short as possible. Maybe even just

```suggestion
**Note:** If this is your first time contributing to Clippy, you should
first read the [Basics docs](doc/basics.md).
```

Reply:
I'm thinking of somewhere in between your two versions:

```suggestion
**Note:** If this is your first time contributing to Clippy, you should
first read the [Basics docs](doc/basics.md). It will help you get your
environment set up.
```